hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -614,6 +615,57 @@ def hpat_pandas_series_sub_number_impl(self, other):
     raise TypingError('{} The object must be a pandas.series or scalar. Given other: {}'.format(_func_name, other))
 
 
+@overload_method(SeriesType, 'sum')
+def hpat_pandas_series_sum(self, axis=None, skipna=None, level=None, numeric_only=None, min_count=0, **kwargs):
+    """
+    Pandas Series method :meth:`pandas.Series.sum` implementation.
+
+    .. only:: developer
+
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_sum1
+    #    Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_sum2
+
+    Parameters
+    ----------
+    self: :class:`pandas.Series`
+        input arg
+    axis:
+        *unsupported*
+    skipna:
+        *unsupported*
+    level:
+        *unsupported*
+    numeric_only:
+        *unsupported*
+    min_count:
+        *unsupported*
+    kwargs:
+        *unsupported*
+
+    Returns
+    -------
+    :obj:`float`
+        scalar or Series (if level specified)
+    """
+
+    _func_name = 'Method sum().'
+
+    if not isinstance(self, SeriesType):
+        raise TypingError('{} The object must be a pandas.series. Given: {}'.format(_func_name, self))
+
+    if axis is not None or skipna is not None or level is not None or numeric_only is not None or min_count != 0:
+        raise TypingError('{} Unsupported parameters. Given axis: {}, skipna: {}, level: {}, numeric_only: {}, min_count {}'
+                          .format(_func_name, axis, skipna, level, numeric_only, min_count))
+
+    def hpat_pandas_series_sum_impl(self):
+        """
+        Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_sum1
+        """
+        return numpy.nansum(self._data)
```

Comment:
It looks like it should be `numpy.sum` if `skipna` parameter is false

Reply:
Done